src/components/CandidateCategoryTile.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import AppIcon from 'cozy-ui/react/AppIcon'
+import IconGrid from 'cozy-ui/react/Labs/IconGrid'
+import AppLinker, { generateWebLink } from 'cozy-ui/react/AppLinker'
+import { withClient } from 'cozy-client'
+
+class CandidateCategoryTile extends React.Component {
+  render() {
+    const { t, slugs, category, client } = this.props
+    const { domain, secure } = this.context
+    const cozyURL = new URL(client.getStackClient().uri)
+    const app = 'store'
+    const nativePath = `/discover?type=konnector&category=${category}`
+
+    return (
+      <AppLinker
+        slug={app}
+        nativePath={nativePath}
+        href={generateWebLink({
+          cozyUrl: cozyURL.origin,
+          slug: app,
+          nativePath: nativePath
+        })}
+      >
+        {({ onClick, href }) => (
+          <a onClick={onClick} href={href} className="item item--ghost">
+            <div className="item-icon">
+              <IconGrid>
+                {slugs.map(slug => (
+                  <AppIcon
+                    alt={t('app.logo.alt', { name: category })}
+                    app={slug}
+                    key={slug}
+                    className="item-grid-icon"
+                    domain={domain}
+                    secure={secure}
```

Comment:
AppIcon should be able to only take a `cozyURL` props (coming from new URL(client.getStackClient().uri) for instance) and not deal with `secure` and `domain` no ?

Reply:
I think so too, I'm not sure why it takes 2 distinct props.